storio-sample-app/src/main/java/com/pushtorefresh/storio2/sample/ui/fragment/TweetsContentResolverFragment.java

```diff
@@ -0,0 +1,255 @@
+                            tweetsAdapter.setTweets(Collections.<Tweet>emptyList());
+                        } else {
+                            uiStateController.setUiStateContent();
+                            tweetsAdapter.setTweets(tweets);
+                        }
+                    }
+                }, new Consumer<Throwable>() {
+                    @Override
+                    public void accept(Throwable throwable) {
+                        // In cases when you are not sure that query will be successful
+                        // You can prevent crash of the application via error handler
+                        Timber.e(throwable, "reloadData()");
+                        uiStateController.setUiStateError();
+                        tweetsAdapter.setTweets(Collections.<Tweet>emptyList());
+                    }
+                });
+        // Preventing memory leak (other Observables: Put, Delete emit result once so memory leak won't live long)
+        // Because rx.Observable from Get Operation is endless (it watches for changes of tables from query)
+        // You can easily create memory leak (in this case you'll leak the Fragment and all it's fields)
+        // So please, PLEASE manage your subscriptions
+        // We suggest same mechanism via storing all subscriptions that you want to unsubscribe
+        // In something like CompositeSubscription and unsubscribe them in appropriate moment of component lifecycle
+        disposeOnStop(disposable);
+    }
+
+    @OnClick(R.id.tweets_empty_ui_add_tweets_button)
+    void addTweets() {
+        final List<Tweet> tweets = new ArrayList<Tweet>();
+
+        tweets.add(Tweet.newTweet("artem_zin", "Checkout StorIO — modern API for SQLiteDatabase & ContentResolver"));
+        tweets.add(Tweet.newTweet("HackerNews", "It's revolution! Dolphins can write news on HackerNews with our new app!"));
+        tweets.add(Tweet.newTweet("AndroidDevReddit", "Awesome library — StorIO"));
+        tweets.add(Tweet.newTweet("Facebook", "Facebook community in Twitter is more popular than Facebook community in Facebook and Instagram!"));
+        tweets.add(Tweet.newTweet("Google", "Android be together not the same: AOSP, AOSP + Google Apps, Samsung Android"));
+        tweets.add(Tweet.newTweet("Reddit", "Now we can send funny gifs directly into your brain via Oculus Rift app!"));
+        tweets.add(Tweet.newTweet("ElonMusk", "Tesla Model S OTA update with Android Auto 5.2, fixes for memory leaks"));
+        tweets.add(Tweet.newTweet("AndroidWeekly", "Special issue #1: StorIO — forget about SQLiteDatabase, ContentResolver APIs, ORMs suck!"));
+        tweets.add(Tweet.newTweet("Apple", "Yosemite update: fixes for Wifi issues, yosemite-wifi-patch#142"));
+
+        // Looks/reads nice, isn't it?
+        storIOContentResolver
+                .put()
+                .objects(tweets)
+                .prepare()
+                .asRxFlowable(MISSING)
```

Comment:
Can be Completable to avoid empty callbacks?

Reply:
Valid point 👍